### ISSUE TITLE
Nextflow error

### ISSUE THREAD

**Issue by francicco** [hillerlab/TOGA#59]
Hi,

I'm trying to test the TOGA installation, and I'm getting this error:

```
nextflow /user/work/tk19812/software/TOGA-1.1.2/execute_joblist.nf --joblist /user/work/tk19812/software/TOGA-1.1.2/micro_test_out/temp/chain_class_jobs_combined
N E X T F L O W  ~  version 23.04.1
Nextflow DSL1 is no longer supported — Update your script to DSL2, or use Nextflow 22.10.x or earlier
```
this is the whole log

```
./run_test.sh micro
Compiling C code...
Model found
CESAR installation not found, cloning
fatal: not a git repository (or any parent up to mount point /user)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
fatal: not a git repository (or any parent up to mount point /user)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
./configure.sh: line 30: cd: CESAR2.0-1.01: No such file or directory
Don't worry about '*** are the same file' message if you see it
Configure successful!
Running test in the micro mode
Calling ./toga.py test_input/align_micro_sample.chain test_input/annot_micro_sample.bed          test_input/hg38.micro_sample.2bit test_input/q2bit_micro_sample.2bit          --pn micro_test_out  --kt          --cjn 1 --chn 1 --ms          
#### Initiating TOGA class ####
Checking dependencies...
check if binaries are compiled and libs are installed...
All dependencies found
fatal: not a git repository (or any parent up to mount point /user)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
Output directory: /user/work/tk19812/software/TOGA-1.1.2/micro_test_out
./modules/chain_score_filter test_input/align_micro_sample.chain 15000 > /user/work/tk19812/software/TOGA-1.1.2/micro_test_out/temp/genome_alignment.chain in progress...
./modules/chain_score_filter test_input/align_micro_sample.chain 15000 > /user/work/tk19812/software/TOGA-1.1.2/micro_test_out/temp/genome_alignment.chain done with code 0
Checking input files correctness...
Continue without isoforms file: not provided
Detected 1 sequences in /user/work/tk19812/software/TOGA-1.1.2/test_input/hg38.micro_sample.2bit
Detected 1 sequences in /user/work/tk19812/software/TOGA-1.1.2/test_input/hg38.micro_sample.2bit
Detected 2 sequences in /user/work/tk19812/software/TOGA-1.1.2/test_input/q2bit_micro_sample.2bit
#### TOGA initiated successfully! ####
#### STEP 0: making chain and bed file indexes

make_indexed in progress...
Reading chain...
There are 3 chains to index
Saved chain index data...
Saved chain /user/work/tk19812/software/TOGA-1.1.2/micro_test_out/temp/genome_alignment.chain index to /user/work/tk19812/software/TOGA-1.1.2/micro_test_out/temp/genome_alignment.bst
Indexed
index_bed in progress...
Making gene_id: bed_data dict...
Bed file indexed
#### STEP 1: Generate extract chain features jobs

./split_chain_jobs.py /user/work/tk19812/software/TOGA-1.1.2/micro_test_out/temp/genome_alignment.chain /user/work/tk19812/software/TOGA-1.1.2/micro_test_out/temp/toga_filt_ref_annot.bed /user/work/tk19812/software/TOGA-1.1.2/micro_test_out/temp/toga_filt_ref_annot.hdf5 --jobs_num 1 --jobs /user/work/tk19812/software/TOGA-1.1.2/micro_test_out/temp/chain_classification_jobs --jobs_file /user/work/tk19812/software/TOGA-1.1.2/micro_test_out/temp/chain_class_jobs_combined --results_dir /user/work/tk19812/software/TOGA-1.1.2/micro_test_out/temp/chain_classification_results --rejected /user/work/tk19812/software/TOGA-1.1.2/micro_test_out/temp/rejected/SPLIT_CHAIN_REJ.txt in progress...
./split_chain_jobs.py /user/work/tk19812/software/TOGA-1.1.2/micro_test_out/temp/genome_alignment.chain /user/work/tk19812/software/TOGA-1.1.2/micro_test_out/temp/toga_filt_ref_annot.bed /user/work/tk19812/software/TOGA-1.1.2/micro_test_out/temp/toga_filt_ref_annot.hdf5 --jobs_num 1 --jobs /user/work/tk19812/software/TOGA-1.1.2/micro_test_out/temp/chain_classification_jobs --jobs_file /user/work/tk19812/software/TOGA-1.1.2/micro_test_out/temp/chain_class_jobs_combined --results_dir /user/work/tk19812/software/TOGA-1.1.2/micro_test_out/temp/chain_classification_results --rejected /user/work/tk19812/software/TOGA-1.1.2/micro_test_out/temp/rejected/SPLIT_CHAIN_REJ.txt done with code 0
#### STEP 2: Extract chain features: parallel step

Extract chain features, project name: micro_test_out_chain_feats_at_1682704341
Project path: ./nextflow_logs/micro_test_out_chain_feats_at_1682704341
Calling nextflow /user/work/tk19812/software/TOGA-1.1.2/execute_joblist.nf --joblist /user/work/tk19812/software/TOGA-1.1.2/micro_test_out/temp/chain_class_jobs_combined
N E X T F L O W  ~  version 23.04.1
Nextflow DSL1 is no longer supported — Update your script to DSL2, or use Nextflow 22.10.x or earlier
Error! Process nextflow /user/work/tk19812/software/TOGA-1.1.2/execute_joblist.nf --joblist /user/work/tk19812/software/TOGA-1.1.2/micro_test_out/temp/chain_class_jobs_combined died
Program finished with exit code 1
```

Thank you
Best
Francesco

**Comment by kirilenkobm** [comment]
Hi @francicco, 

A quick fix would be to downgrade Nextflow to 22.10.4 which still supports DSL1.
Upgrading the TOGA pipeline to support DSL2 may take a while...

Best,
Bogdan

**Comment by francicco** [comment]
Cool! Thanks a lot!
F
